R/auto_tweet.R

```diff
@@ -80,18 +110,27 @@ action_auto_tweet <- function(twitter_token = ser_token, google_drive_auth = dri
   googledrive::drive_download(retweet_csv_id, type = "csv", overwrite = TRUE)
   retweet_queue <- readr::read_csv("retweet_queue.csv", col_types = "cciil")
 
-  # don't post on weekends
-  todays_date <- lubridate::wday(Sys.Date(), label = TRUE) %>%
-    as.character()
+  # don't post on weekends (unless apply_test_date == TRUE)
+  if (apply_test_date) {
+    todays_date <- "Mon"
+  } else {
+    todays_date <- lubridate::wday(Sys.Date(), label = TRUE) %>%
+      as.character()
+  }
+
   if (todays_date %in% c("Mon", "Tue", "Wed", "Thu", "Fri")) {
-    post_tweet_of_type <- post_tweet_queue
+    post_tweet_of_type <- post_tweet_library(tweet_data = tweet_library,
+                                             past_tweets = tweet_hist_list,
+                                             twitter_token = twitter_token)
   } else {
     post_tweet_of_type <- blackout_tweet
```

Comment:
If `post_tweet_of_type` is no longer a function, you need to actually run `blackout_tweet`, e.g.

`post_tweet_of_type <- blackout_tweet()`

Reply:
This also needs to happen on line 143